ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -383,15 +386,21 @@ def _parse_data_model(self):
                 model_name = (
                     mnode["alias"] if "alias" in mnode.keys() else mnode["name"]
                 )
+                model_name = model_name.replace(".", "_DOT_")
                 description = mnode.get("description", "")
                 schema = mnode["schema"]
                 path = f"{mnode['root_path']}/{mnode['original_file_path']}"
+                raw_sql = mnode["raw_sql"] if "raw_sql" in mnode else ""
+                compiled_sql = (
+                    mnode["compiled_sql"] if "compiled_sql" in mnode else raw_sql
+                )
+
                 model = DataModel(
                     modelType=ModelType.DBT,
                     description=description,
                     path=path,
-                    rawSql=mnode["raw_sql"] if "raw_sql" in mnode else None,
-                    sql=mnode["compiled_sql"] if "compiled_sql" in mnode else None,
```

Comment:
it actually feels cleaner supporting the `None`, we might want to revisit the `DataModel` processing and have `raw_sql=mnode.get("raw_sql")` and `sql=mnode.get("compiled_sql", raw_sql)`